modules/ossm-cr-general.adoc

```diff
@@ -0,0 +1,60 @@
+// Module included in the following assemblies:
+//
+// * service_mesh/v2x/ossm-reference-smcp.adoc
+
+[id="ossm-cr-general_{context}"]
+= general parameters
+
+Here is an example that illustrates the `spec.general` parameters for the `ServiceMeshControlPlane` and a description of the available parameters with appropriate values.
```

Comment:
```suggestion
Here is an example that illustrates the `spec.general` parameters for the `ServiceMeshControlPlane` object, and a description of the available parameters with appropriate values.
```
I think it should be either `ServiceMeshControlPlane` object (or resource) or Service Mesh control plane, as per the [guidelines](https://github.com/openshift/openshift-docs/blob/main/contributing_to_docs/doc_guidelines.adoc#api-object-formatting)